### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/doppio/commit/2d189dfebe5741f82440a57163703b4f6ff17eb7#commitcomment-90914884

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/doppio/commit/c947331525a5eb4226033b6fd860c0e21935e3f2

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: classes/demo/Chatterbot.java [hunk 1]
```diff
@@ -471,7 +471,7 @@ public class Chatterbot {
 			}
 		};
 
-	private static String transposList[][] = {
+	private static String transposeList[][] = {
```
Comment (jsoref) [inline classes/demo/Chatterbot.java:474]: I'm pretty sure this is right...

File: classes/test/NashornTest.java [hunk 2]
```diff
@@ -18,7 +18,7 @@ public static void main(String[] args) throws Exception {
     engine.eval("print('Hello World!');");
 
     // JVM -> JS
-    engine.eval("var fun1 = function(name) {\nprint('Hi there from Javascript, ' + name);\nreturn 'greetings from javascript';};\nvar fun2 = function (object) {\nprint('JS Class Definition: ' + Object.prototype.toString.call(object));\n};");
+    engine.eval("var fun1 = function(name) {\nprint('Hi there from JavaScript, ' + name);\nreturn 'greetings from javascript';};\nvar fun2 = function (object) {\nprint('JS Class Definition: ' + Object.prototype.toString.call(object));\n};");
```
Comment (jsoref) [inline classes/test/NashornTest.java:21]: brand

File: classes/test/Reflection.java [hunk 3]
```diff
@@ -58,7 +58,7 @@ public static void main(String[] args)
     Class<Reflection> c = Reflection.class;
 
     // test the ability to reflectively get constants, which are usually baked
-    // in to bytecodes by the compiler
+    // into bytecodes by the compiler
```
Comment (jsoref) [inline classes/test/Reflection.java:61]: This has been the most controversial type of change I've experimented with...

**Review by jsoref — COMMENTED** [review]
Most corrections suggested by Google Sheets. All fault mine...

**Comment by emeryberger** [comment]
Thanks for bringing this service to my attention, but I don't have time to audit this to make sure it doesn't introduce breakage or confusion.

**Pull request closed without merging**